### PR TITLE
[NSO-1178] add imagemagick to the pre-installed packages list

### PR DESCRIPTION
**Pull request by aheadley**
From https://github.com/nexcess/ansible-role-server/blob/master/defaults/main.yml#L52

Not ready for merge, needs to be tested

### CODE REVIEW

File: playbooks/group_vars/cloudhost.yml [hunk 1]
```diff
@@ -31,3 +31,7 @@ firewall_v6_group_rules:
 
 ## Prevent puppet from running in a non-blocking fashion
 puppet_agent_fire_and_forget: false
+
+## Include ImageMagick in the default cloudhost install
+server_extra_packages:
+  - ImageMagick
```
Comment (clwells) [inline playbooks/group_vars/cloudhost.yml]: Please quote "ImageMagick"

**Comment by aheadley** [comment]
@clwells ImageMagick quoted

**Comment by aheadley** [comment]
Change tested fine, but this is on hold pending decision in https://nexcess.atlassian.net/browse/NSO-1178

**Comment by aheadley** [comment]
Closing because we are going to use 6.9 instead.

**Pull request closed without merging**